src/javascript/app_2/App/Components/Elements/ErrorBox/__tests__/error_box.spec.js

```diff
@@ -0,0 +1,19 @@
+import React                  from 'react';
+import { expect }             from 'chai';
+import { configure, shallow } from 'enzyme';
+import Adapter                from 'enzyme-adapter-react-16';
+import ErrorBox               from '../error_box.jsx';
+
+
+configure({ adapter: new Adapter() });
+
+describe('ErrorBox', () => {
+    it('should render one <ErrorBox /> component', () => {
+        const wrapper = shallow(<ErrorBox />);
+        expect(wrapper).to.have.length(1);
+    });
+    it('should render header as passed to it', () => {
+        const wrapper = shallow(<ErrorBox header='This is a header' />);
+        expect(wrapper.find('.page-error-header').text()).to.be.eql('This is a header');
+    });
```

Comment:
I think we should test the children rendering as well. Like what you did in `settings_control.spec.js`. Or even refactor that test into a general function. So any components that contains children can be tested with that function.